_posts/2017-10-16-windows-retrospect.md

```diff
@@ -1,6 +1,6 @@
 ---
 layout: posts
-title: Bazel on Windows: a year in retrospect
+title: "Bazel on Windows: a year in retrospect"
```

Comment:
Please change to:

```
title: "Bazel on Windows -- a year in retrospect"
```

The ":" in the middle confuses the Markdown renderer. I had to edit it when I rendered the pages locally.

Reply:
This is the reason for the quotes. The repository does not contains --, do you mean that `--` should be on the repository?

Reply:
So you say the quotes make Jekyll happy? OK then.

I don't understand what you mean with the second sentence.